common.blocks/form-field/_type/form-field_type_hidden.browser.js

```diff
@@ -0,0 +1,18 @@
+modules.define('form-field', function(provide, FormField) {
+
+    provide(FormField.decl({ modName : 'type', modVal : 'hidden' }, {
+
+        getName : function() {
+            return this.params.name;
+        },
+
+        setVal : function(val) {
+            this.params.value = val;
```

Comment:
By `DOM` you mean that the block should render an `<input type="hidden" value="..." name="...">` ? I thought about that, but what difference does it make?
Also, what you mean by *values on server side*?